core-bundle/src/Cron/Cron.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Cron;
+
+use Contao\CoreBundle\Entity\Cron as CronEntity;
+use Contao\CoreBundle\Repository\CronRepository;
+use Cron\CronExpression;
+use Psr\Log\LoggerInterface;
+
+class Cron
+{
+    /**
+     * @var string
```

Comment:
Nitpicking: I don't think constants ever have docblocks

Reply:
But constants can be strings, floats, integers or arrays - shouldn't this be defined this way?